src/styles/main.scss

```diff
@@ -1,3 +1,239 @@
+html {
+  font-family: "Roboto", "Helvetica", sans-serif;
+}
+
+* {
+  box-sizing: border-box;
+  margin: 0;
+}
+
 body {
   margin: 0;
 }
+
+.container {
+  padding-top: 50px;
+  padding-bottom: 72px;
+  display: grid;
+  grid-template-columns: repeat(1, 200px);
+  row-gap: 46px;
+  column-gap: 48px;
+  justify-content: center;
+
+  @media (min-width: 488px) {
+    & {
+      grid-template-columns: repeat(2, 200px);
+    }
+  }
+
+  @media (min-width: 768px) {
+    & {
+      grid-template-columns: repeat(3, 200px);
+    }
+  }
+
+  @media (min-width: 1024px) {
+    & {
+      grid-template-columns: repeat(4, 200px);
+    }
+  }
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px ;
+  --text-color: #060b35;
+  --active-color: #00acdc;
+  font-family: "Roboto", sans-serif;
```

Comment:
You already set this as default font for the page. You don't need to duplicate this